Microsoft.Toolkit.Uwp.UI.Controls.Core/Microsoft.Toolkit.Uwp.UI.Controls.Core.csproj

```diff
@@ -1,10 +1,14 @@
 <Project Sdk="MSBuild.Sdk.Extras">
 
   <PropertyGroup>
-    <TargetFrameworks>uap10.0.17763</TargetFrameworks>
-    <Title>Windows Community Toolkit Controls</Title>
+    <TargetFramework>uap10.0.17763</TargetFramework>
+    <RootNamespace>Microsoft.Toolkit.Uwp.UI.Controls</RootNamespace>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Title>Windows Community Toolkit - Common Controls</Title>
```

Comment:
```suggestion
    <Title>Windows Community Toolkit - Core Controls</Title>
```

Reply:
The term `Core` usually mean `Base`/`Inside` of something. Should've been `Common`. It is too late to change the Package ID, but we'll use the correct term in the title.